sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdClientChannelHealthChecker.java

```diff
@@ -4,18 +4,14 @@
 package com.azure.cosmos.implementation.directconnectivity.rntbd;
```

Comment:
NOTE TO REVIEWERS
The changes in `RntbdClientChannelHealthChecker` adds javadocs and simplifies the code a little by eliminating `RntbdClientChannelHealthChecker.JsonSerializer` and breaking code lines for readability and compliance with Azure Central SDK code guidelines.